x-pack/plugins/siem/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -6,23 +6,49 @@
 
 import { shallow } from 'enzyme';
 import toJson from 'enzyme-to-json';
+import { cloneDeep } from 'lodash/fp';
 import * as React from 'react';
+import { MockedProvider } from 'react-apollo/test-utils';
+import { render } from 'react-testing-library';
 import { mountWithIntl } from 'test_utils/enzyme_helpers';
 
+import { mockFirstLastSeenHostQuery } from '../../../../containers/hosts/first_last_seen/mock';
+import { wait } from '../../../../lib/helpers';
 import { TestProviders } from '../../../../mock';
 import { getEmptyString, getEmptyValue } from '../../../empty_value';
 
 import { createDraggable, getEuiDescriptionList, HostSummary } from './index';
 import { mockData } from './mock';
 
 describe('Host Summary Component', () => {
+  // this is just a little hack to silence a warning that we'll get until react
+  // fixes this: https://github.com/facebook/react/pull/14853
```

Comment:
👍 Thanks for the PR reference and the exact version number here for when to remove this